app/vmctl/vm_native.go

```diff
@@ -89,6 +89,7 @@ func (p *vmNativeProcessor) run(ctx context.Context) error {
 		log.Printf("error start process bars pool: %s", err)
 		return err
 	}
+	defer barpool.Stop()
```

Comment:
Shall we check the rest of migration modes in case they have the similar error?

Reply:
Sure, will do.